gapic/src/main/com/google/gapid/views/ThumbnailScrubber.java

```diff
@@ -343,10 +373,12 @@ public void paint(BigInteger xOffset, BigInteger yOffset, GC gc) {
               imageSize =
                   new Point(Math.max(MIN_SIZE, bounds.width), Math.max(MIN_SIZE, bounds.height));
               updateSize.run();
+              selectAndScroll(selectedIndex);
```

Comment:
We update the state / prepare the state of the next frame in the animation. I don't think it would make a difference if it was done outside of paint, as it, too, would have to run on the UI thread.